notification/cel.go

```diff
@@ -85,6 +86,10 @@ func (t *celVariables) AsMap() map[string]any {
 		"author":  lo.FromPtr(t.Author).AsMap(),
 	}
 
+	if v, err := lo.FromPtr(t.ConfigItem).TemplateEnv(); err == nil {
```

Comment:
Shouldn't AsMap just be a call to TemplateEnv() with the removal of fields?

Reply:
I'm not sure if the fields we were removing should be removed anymore. The `path` could be useful for eg.

`TemplateEnv` could even go away and `AsMap` just returns the map on `.config`